### PR TITLE
chore: add tables sql script

### PR DESCRIPTION
**Pull request by geolffreym**
- Entity Relation design proposal for tables

### CODE REVIEW

**Comment by geolffreym** [comment]
Screenshot from 2022-08-01 12-27-28

File: src/core/cache/tables.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+-- Tables schema definition for watchit cache storage.
+-- ref: https://www.sqlstyle.guide/
+-- Schema table definition for movies.
+CREATE TABLE IF NOT EXISTS movies (
+    movie_id INTEGER PRIMARY KEY,
+    -- imdb code is adopted from IMB movies site to handle an alpha-numeric id
+    imdb_code TEXT KEY DESC,
```
Comment (aphelionz) [inline src/core/cache/tables.sql:7]: if these identifiers are fixed length you may consider VARCHAR

**Review by aphelionz — APPROVED** [review]
Approved contingent on one more pass of SQL types for each of these, particularly the identifiers

File: src/core/cache/tables.sql (hunk 1 above)
Comment (geolffreym) [reply src/core/cache/tables.sql:7]: sqlite doesnt support it :( 
https://sqlite.org/matrix/datatype3.html

**Pull request merged**